app/content/preload/content.js

```diff
@@ -12,4 +12,7 @@ specific language governing permissions and limitations under the License.
 
 import './hover-status';
 import './context-menu';
+import { readerify } from './reader';
 import './scroll';
+
+window._readerify = readerify;
```

Comment:
This isn't hidden from content, right?  Is it possible to hide this from content?